tools/repl/swift/REPLTime/REPLTime.h

```diff
@@ -0,0 +1,38 @@
+// REPLTime.h
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+// -----------------------------------------------------------------------------
+
+#ifndef REPL_TIME_H
+#define REPL_TIME_H
+
+typedef unsigned long long uint64_t;
```

Comment:
Ah, redefining these types is likely to cause problems; if you really need to use them, you should name them something different. In retrospect that means this isn't going to work because you can't redefine the real `mach_absolute_time` in the header; you'd have to make it a mixed-source target (i.e. compile in a .c file to forward the calls for you).